### ISSUE TITLE
no such column: blocking

### ISSUE THREAD

**Issue by eserte** [jhthorsen/devel-mojoprof#1]
On some of my smoker systems (without a recognizable pattern so far) t/sqlite.t fails:
```
DBD::SQLite::db prepare_cached failed: no such column: blocking at /usr/home/cpansand/.cpan/build/2019110607/Devel-MojoProf-0.03-2/blib/lib/Devel/MojoProf.pm line 71.
t/sqlite.t .... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by eserte** [comment]
It could be related to newer versions of Mojo::SQLite and/or DBD::SQLite. However, there's no statistical analysis for this assumption.

**Comment by jhthorsen** [comment]
This has totally slipped my mind! Guess this module doesn't have too many users...

Fixed in 0.05.
